spec/features/state_file/complete_intake_spec.rb

```diff
@@ -695,34 +622,37 @@
         visit "/"
         click_on "Start Test NY"
 
-        expect(page).to have_text I18n.t("state_file.landing_page.ny_closed.title")
+        page_change_check(I18n.t("state_file.landing_page.ny_closed.title"))
 
         # spot check a few pages for redirecting to landing page
         visit "/questions/ny-eligibility-residence"
-        expect(page).to have_text I18n.t("state_file.landing_page.ny_closed.title")
+        page_change_check(I18n.t("state_file.landing_page.ny_closed.title"))
 
         visit "/questions/ny-sales-use-tax"
-        expect(page).to have_text I18n.t("state_file.landing_page.ny_closed.title")
+        page_change_check(I18n.t("state_file.landing_page.ny_closed.title"))
 
         visit "/questions/ny-review"
-        expect(page).to have_text I18n.t("state_file.landing_page.ny_closed.title")
+        page_change_check(I18n.t("state_file.landing_page.ny_closed.title"))
 
         # try to log in with existing ny intake and get redirected
         visit "/questions/return-status"
-        expect(page).to have_text "Sign in to FileYourStateTaxes"
+        page_change_check("Sign in to FileYourStateTaxes")
         click_on "Sign in with email"
-        expect(page).to have_text "Sign in with your email address"
+        page_change_check("Sign in with your email address")
         fill_in "Your email address", with: email_address
         perform_enqueued_jobs do
           click_on "Send code"
         end
-        expect(page).to have_text "Enter the code to continue"
+
+        page_change_check("Enter the code to continue")
         fill_in "Enter the 6-digit code", with: verification_code
         click_on "Verify code"
-        expect(page).to have_text I18n.t("state_file.intake_logins.edit.title")
+
+        page_change_check(I18n.t("state_file.intake_logins.edit.title"))
```

Comment:
this unfortunately remains to be flakey because it'll register "000004" as the incorrect verification code every once and a while; not sure if this is a job problem; i tried trying to click the verify button again if the "incorrect code" text was present but no dice